pulp_container/app/models.py

```diff
@@ -199,7 +262,12 @@ class ContainerNamespace(BaseModel, AutoAddObjPermsMixin, AutoDeleteObjPermsMixi
     class Meta:
         unique_together = (("name",),)
         permissions = [
-            ("manage_namespace_distributions", "Can manage distributions in a namespace"),
+            ("namespace_add_containerdistribution", "Add a distribution to a namespace"),
+            ("namespace_delete_containerdistribution", "Delete a distribution from a namespace"),
+            ("namespace_view_containerdistribution", "View a distribution in a namespace"),
+            ("namespace_pull_containerdistribution", "Pull from all distributions in a namespace"),
+            ("namespace_push_containerdistribution", "Push to all distributions in a namespace"),
+            ("namespace_change_containerdistribution", "Change all distributions in a namespace"),
```

Comment:
:+1:

Reply:
unfortunately, it's the maximum recursive depth for the thumb-ups :disappointed: